### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by littlefico** [timokoessler/2FAGuard#51] [Ideas]
Hi Timo,
you got a very nice programm i use from now on.
would it be possible to sort the tokens into folders or something like that?
so it would be more organized.
i have about 20+ tokens and have to search for the right one.

**Comment by timokoessler** [comment]
Hi,
Thanks! Wouldn't tags (#45) solve this problem too?
Tags would also allow adding one token to multiple "groups".

**Reply by littlefico** [reply]
im not sure if this would look like this.
sorry for the amateur editing.
![IMG_20241124_103151_new](https://github.com/user-attachments/assets/32725609-00a0-4995-ba6d-bfbf74b1027b)

id like to have some sort of choice to show all at once (like it is now) or into groups i can create and rename as i like.
if that is like the tag version at #45 im okay with that.

**Reply by timokoessler** [reply]
I think this will be more or less possible with the tag feature.

**Reply by littlefico** [reply]
thx man,
then plz make it possible too to tick one of those folders/tags to show just the selected one.
but also to show all the tags/folders one the whole screen (tickbox on the upper side of my pic)

**Reply by littlefico** [reply]
is the tag feature in process?
